### PR TITLE
React/react1/week1

### PR DESCRIPTION
**Pull request by juliadavydenko**
Basic todo list with some styling and pics.
Used Cards. Removed a couple of files from the starting project pack. Added instead cards.jsx, styles.css and tasks.js file.

### CODE REVIEW

File: react/react1/week1/React - ONE/vite-project/src/App.jsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+import React from "react";
+import ReactDOM from "react-dom";
+import Card from "./Card";
+import tasks from "./tasks.js";
+function App() {
+  return (
+    <div>
+      <h1 className="heading">ToDo LIST</h1>
+      <Card
```
Comment (LucasIversen) [inline react/react1/week1/React - ONE/vite-project/src/App.jsx:14]: Good use of component here 👍

File: react/react1/week1/React - ONE/vite-project/src/Card.jsx [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import React from "react";
+
```
Comment (LucasIversen) [inline react/react1/week1/React - ONE/vite-project/src/Card.jsx:2]: Perfect. Props used correctly, and awesome to see you do some styling also 🥇

**Review by LucasIversen — COMMENTED** [review]
Hey Julia. Really good job on your homework! keep it up 💪

File: react/react1/week1/React - ONE/vite-project/src/Card.jsx (hunk 2 above)
Comment (juliadavydenko) [reply react/react1/week1/React - ONE/vite-project/src/Card.jsx:2]: @LucasIversen Thank you! I will try to keep this up and explore React more :)

**Pull request merged**